sensor_msgs/Image image
---
mav_msgs/RateThrust rateThrust
